lib/src/models/Cell.dart

```diff
@@ -23,13 +29,42 @@ class Cell extends ICell {
 
   /// Constructs new Cell at [position] with optional [_value]
   /// If value is provided, then cell is flagged as prefilled
-  Cell(position, [this._value = 0]) : super(position) {
+  Cell(this.position, [this._value = 0]){
```

Comment:
```suggestion
  Cell(position, [this._value = 0]) : super(position, true){
```

_Part of the 'let's keep ICell' movement ;)_